inputremapper/injection/macros/macro.py

```diff
@@ -469,10 +469,11 @@ def add_event(self, type_: Union[str, int], code: Union[str, int], value: int):
         self.tasks.append(lambda handler: handler(type_, code, value))
         self.tasks.append(self._keycode_pause)
 
-    def add_mouse(self, direction: str, speed: int):
+    def add_mouse(self, direction: str, speed: int, accel: Optional[float] = None):
```

Comment:
I'd prefer `accel` to be written out as `acceleration`

Reply:
Done.